qa/tasks/mgr/dashboard/test_rgw.py

```diff
@@ -245,14 +245,15 @@ def test_create_get_update_delete_w_tenant(self):
             })
         self.assertStatus(200)
         data = self._get('/api/rgw/bucket/{}'.format(
-            urllib.quote_plus('testx/teuth-test-bucket')))
+            urllib.quote_plus('teuth-test-bucket')))
```

Comment:
You should remove ``urllib.quote_plus``here because it was only used to quote the ``/``character. The method has been removed in Python 3.x, too.

Reply:
Will be fixed in https://tracker.ceph.com/issues/41180

Reply:
> You should remove `urllib.quote_plus`here because it was only used to quote the `/`character. The method has been removed in [Python 3.x](https://docs.python.org/3.5/library/urllib.html), too.

Will be fixed in https://tracker.ceph.com/issues/41180